agent/dockerclient/dockerapi/docker_client_test.go

```diff
@@ -731,7 +736,7 @@ func TestContainerEventsStreamError(t *testing.T) {
 
 	eventsChan := make(chan events.Message, dockerEventBufferSize)
 	errChan := make(chan error)
-	mockDockerSDK.EXPECT().Events(gomock.Any(), gomock.Any()).Return(eventsChan, errChan).Times(2)
+	mockDockerSDK.EXPECT().Events(gomock.Any(), gomock.Any()).Return(eventsChan, errChan).MinTimes(1)
```

Comment:
for my curiosity, why did you loosen the required call count check?

Reply:
so looking at this test `TestContainerEventsStreamError` we only pass up a single event to test this codepath. so we should we expecting at least one call to `Events(...)`.

where as with the `Times(2)`, we would sometimes hit the `Events(...)` call on the first iteration through this [loop](https://github.com/aws/amazon-ecs-agent/blob/dev/agent/dockerclient/dockerapi/docker_client.go#L864) and would get test failures for missing calls to the mock from time to time. for example running with a `-count=100` you'll likely see test failure every time. 

let me know if you think i've missed something -  i think we added this test as part of the moby migration yea?